guides/cli/how-to-use-lightdash-deploy.mdx

```diff
@@ -5,23 +5,77 @@ icon: "ship"
 ---
 
 <Callout icon="ship" color="#30CF87">
-### `lightdash deploy` pushes your dbt project to Lightdash
+### `lightdash deploy`pushes your dbt project to Lightdash
 </Callout>
 
-If you've made some changes to your dbt project and you'd like to make them available in your Lightdash project, you can easily do this using the `lightdash deploy` command in the Lightdash CLI tool.
+Once you've made updates to your dbt project and you're ready to make those changes available in your Lightdash production project, you have a few options for deploying from the CLI or the Lightdash UI. 
+This guide will walk you through how it works, when to use each option, and what to keep in mind for a safe and scalable workflow.
 
+## Before you deploy: A quick word of caution
+
+<Warning>
+Running `lightdash deploy` uses your local dbt profile (from your `profiles.yml`). If your local profile isn't pointing to production, you could deploy to the wrong environment. 
+This works in simple setups, but can introduce risk when using multiple environments or working in a team.
+</Warning>
+
+For a safer, scalable approach to deploying and previewing changes, use CI/CD. This allows you to:
+
+* Spin up Lightdash preview environments for each PR
+
+* Deploy production changes consistently using shared, secure credentials
+
+* Avoid accidentally pointing to the wrong target in your local profile
+
+Check out our [**guide to setting up CI/CD workflow**](/guides/cli/automate-cli-workflow)
+
+## Option 1: Deploy via the CLI
+
+Basic usage:
 ```bash
 git checkout main # checkout main or master - or whatever your production branch name is where you've merged your changes
 git pull
 lightdash deploy # --target prod. If you use developer profiles in your dbt project, you might need this flag. See below.
 ```
 
-This will deploy the changes in your dbt project to the Lightdash project you set up on your CLI tool earlier.
+By default, this will:
+
+* Use your local dbt profile for the connection
+
+* Push your dbt project to the Lightdash project you've authenticated with via the CLI
```

Comment:
@torimwhaley i think there needs to be a small clarification here that the "dbt project" getting pushed is whatever is currently selected in your local environment. For example, if you are on a branch, it will deploy the code from the branch, if you have local changes saved that aren't even committed to git yet, those will be deployed.